avro/src/main/java/com/fasterxml/jackson/dataformat/avro/schema/VisitorFormatWrapperImpl.java

```diff
@@ -49,6 +49,10 @@ public VisitorFormatWrapperImpl(DefinedSchemas schemas, SerializerProvider p) {
         _provider = p;
     }
 
+    protected VisitorFormatWrapperImpl createVisitorWrapper() {
```

Comment:
This seems odd from naming perspective: type with method to create what seems like (but isn't) a copy.

Would it conceptually be more somethnig like "createChildWrapper()" or "createChildInstance()" or something?

Reply:
I have chosen `createChildWrapper`. Did not find better name.